litex/build/lattice/radiant.py

```diff
@@ -153,7 +153,9 @@ def tcl_path(path): return path.replace("\\", "/")
 
         # Add include paths
         vincpath = ";".join(map(lambda x: tcl_path(x), self.platform.verilog_include_paths))
-        tcl.append("prj_set_impl_opt {include path} {\"" + vincpath + "\"}")
+        if vincpath != "": # Do not add quotes if vincpath is empty, else Radiant complains
```

Comment:
An empty string is seen as `False` so:
````
if vincpath and vincpath.strip():
   [...]
```
is more pythoniic and catch empty string and string with only blank.